ch_tools/chadmin/cli/data_store_group.py

```diff
@@ -30,40 +33,89 @@ def data_store_group():
     default=False,
     help="Flag to REMOVE data from store subdirectories.",
 )
-def clean_orphaned_tables_command(column, remove):
-    for prefix in os.listdir(CLICKHOUSE_STORE_PATH):
-        path = CLICKHOUSE_STORE_PATH + "/" + prefix
-
-        process_path(path, prefix, column, remove)
-
-
-def process_path(path: str, prefix: str, column: str, remove: bool) -> None:
-    print(f"Processing path {path} with prefix {prefix}:")
+@option(
+    "--store-path",
+    "store_path",
+    default=CLICKHOUSE_PATH + "/store",
+    help="Set the store subdirectory path,  `/var/lib/clickhouse/store` by default.",
```

Comment:
I think we can remove `by default` clause here. Because we already show default values by this setting https://github.com/yandex/ch-tools/blob/6b9c5d2c441f61a7c01c9ab82768f55bec2cb6b2/ch_tools/common/cli/context_settings.py#L13